src/path.cpp

```diff
@@ -0,0 +1,110 @@
+#include "path.hpp"
+
+#ifdef _WIN32
+
+#if defined(SENTRY_BREAKPAD)
+#include "common/string_conversion.h"
+
+static std::wstring cstr_to_wtr(const char *s) {
+    std::vector<uint16_t> vec;
+    google_breakpad::UTF8ToUTF16(s, vec);
+    return std::wstring(vec.cbegin(), vec.cend());
+}
+
+#elif defined(SENTRY_CRASHPAD)
+#include "base/strings/utf_string_conversions.h"
+
+static std::wstring cstr_to_wstr(const char *s) {
+    std::wstring rv;
+    base::UTF8ToUTF16(s, strlen(s), &rv);
+    return rv;
+}
+#endif
+#endif
+
+namespace sentry {
+#ifdef _WIN32
+Path::Path(const char *path) : m_path(cstr_to_wstr(path)) {
+}
+
+Path Path::join(const wchar_t *other) const {
+    if (::isalpha(*other) && other[1] == L':') {
```

Comment:
Maybe check for the length of `other`, just to be sure, same for `m_path`.

Reply:
I don't see the reason for doing that